digServiceExtractor/names_helper.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-10-05 16:01:52
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-10-06 22:40:34
+
+import os
+import json
+import string
+import pygtrie as trie
+from digDictionaryExtractor.populate_trie import populate_trie
+from digDictionaryExtractor.name_dictionary_extractor import get_name_dictionary_extractor
+from digExtractor.extractor_processor import ExtractorProcessor
+from digCrfTokenizer.crf_tokenizer import CrfTokenizer
+
+def load_names(path=os.path.join(os.path.dirname(__file__), 'names.json')):
```

Comment:
If we have a set of default names we always use, we should include them in a python module as an array instead of loading from disk and parsing it.  Then the user should have the option of overriding or adding to the trie later.  I don't think this will work as written when we publish it to pypi because setup tools won't include names.json in the egg/wheel.